code/modules/reagents/reagent_containers/cups/_cup.dm

```diff
@@ -391,7 +389,7 @@
 			if(reagents.total_volume == reagents.maximum_volume)
 				to_chat(user, "<span class='warning'>[src] is full!</span>")
 				return
-			O.reagents.remove_any(O.reagents.total_volume*SQUEEZING_DISPERSAL_PERCENT)
+			O.reagents.remove_any(O.reagents.total_volume * SQUEEZING_DISPERSAL_RATIO)
```

Comment:
also undocced

Reply:
This is Skyrat stuff already, if you look up.

Reply:
Looks like I didn't look high enough then